src/main/java/org/archive/hadoop/jobs/WEATGenerator.java

```diff
@@ -93,17 +97,25 @@ public void map( Text key, Text value, OutputCollector output, Reporter reporter
         String inputBasename = inputPath.getName();
         String watOutputBasename = "";
         String wetOutputBasename = "";
+        String cdxWatOutputBasename = "";
+        String cdxWetOutputBasename = "";
 
         if(path.endsWith(".gz")) {
           watOutputBasename = inputBasename.substring(0,inputBasename.length()-3) + ".wat.gz";
           wetOutputBasename = inputBasename.substring(0,inputBasename.length()-3) + ".wet.gz";
+          cdxWatOutputBasename = inputBasename.substring(0,inputBasename.length()-3) + ".cdxwat.gz";
```

Comment:
This results in:
```
name.warc.gz      name.cdx.gz
name.warc.wat.gz  name.warc.wat.cdxwat.gz
name.warc.wet.gz  name.warc.wet.cdxwet.gz
```

- "wat" is given twice
- ".warc" is removed for CDX files derived from WARC files
  - should be the same for WAT/WET files
  - a CDX file does not follow the WARC format
  - (a WAT or WET file does)

Maybe the following looks better?
```
name.warc.gz      name.cdx.gz
name.warc.wat.gz  name.wat.cdx.gz
name.warc.wet.gz  name.wet.cdx.gz
```